src/index.html

```diff
@@ -1,13 +1,322 @@
+
+        <div class="now-on-view__wrapper wrapper">
+          <article class="wrapper__article ">
+            <div class="photos">
+              <a href="#" class="photos__link photos__link--wide">
+                <picture>
+                  <source media="(min-width:744px)" srcset="./images/sections/now-on-view/now-on-view-1-tablet.png">
+                  <source media="(min-width:1024px)" srcset="./images/sections/now-on-view/now-on-view-1-desktop.png">
+                  <img
+                    src="./images/sections/now-on-view/now-on-view-1-mobile.png"
+                    alt="collection"
+                    class="photos__image image"
+                  >
+                </picture>
+              </a>
+
+              <a href="#" class="photos__link photos__link--normal">
+                <img
+                  src="./images/sections/now-on-view/now-on-view-2-2x.png"
+                  alt="woman"
+                  class="photos__image image"
+                >
+              </a>
+            </div>
+
+            <h3 class="wrapper__article-title">A New Look at Old Masters</h3>
+
+            <p class="wrapper__article-text">December 12, 2020 - Through Spring 2023</p>
+          </article>
+
+          <article class="wrapper__article ">
+            <div class="photos photos--triple">
+              <a href="#" class="photos__link photos__link--square">
+                <img
+                  src="./images/sections/now-on-view/now-on-view-3-2x.png"
+                  alt="indian god"
+                  class="photos__image image"
+                >
+              </a>
+              <a href="#" class="photos__link photos__link--square">
+                <img
+                  src="./images/sections/now-on-view/now-on-view-4-2x.png"
+                  alt="carpet"
+                  class="photos__image image"
+                >
+              </a>
+
+              <a href="#" class="photos__link photos__link--hidden photos__link--square">
+                <img
+                  src="./images/sections/now-on-view/now-on-view-5-2x.png"
+                  alt="beatiful pattern"
+                  class="photos__image image"
+                >
+              </a>
+            </div>
+
+            <h3 class="wrapper__article-title">Bodhisattvas of Wisdom, Compassion, and Power</h3>
+
+            <p class="wrapper__article-text">March 27, 2021 - October 30, 2022</p>
+          </article>
+
+          <article class="wrapper__article ">
+            <div class="photos">
+              <a href="#" class="photos__link photos__link--normal">
+                <img
+                  src="./images/sections/now-on-view/now-on-view-6-2x.png"
+                  alt="mystical creatures"
+                  class="photos__image image"
+                >
+              </a>
+
+              <a href="#" class="photos__link photos__link--wide">
+                <img src="./images/sections/now-on-view/now-on-view-7-2x.png"
```

Comment:
Look at the image on mobile resolution and fix object position

Reply:
fixed